tools/rule-notices.ts

```diff
@@ -0,0 +1,118 @@
+import plugin from '../src';
+
+enum MESSAGE_TYPE {
+  CONFIGS = 'configs',
+  DEPRECATED = 'deprecated',
+  FIXABLE = 'fixable',
+  HAS_SUGGESTIONS = 'hasSuggestions',
+}
+
+export const MESSAGES = {
+  [MESSAGE_TYPE.CONFIGS]:
+    '💼 This rule is enabled in the following [configs](https://github.com/jest-community/eslint-plugin-jest/blob/main/README.md#shareable-configurations):',
+  [MESSAGE_TYPE.DEPRECATED]: '❌ This rule is deprecated.',
+  [MESSAGE_TYPE.FIXABLE]:
+    '🔧 This rule is automatically fixable using the `--fix` [option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix) on the command line.',
+  [MESSAGE_TYPE.HAS_SUGGESTIONS]:
+    '💡 This rule provides [suggestions](https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions) that can be applied manually.',
+};
+
+type Rule = typeof plugin.rules[string];
+
+/**
+ * Get config names that a given rule belongs to.
+ */
+function getConfigsForRule(ruleName: string) {
+  const { configs } = plugin;
+  const configNames: Array<keyof typeof configs> = [];
+  let configName: keyof typeof configs;
+
+  for (configName in configs) {
+    const config = configs[configName];
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment -- we don't have a static type for rule names
+    // @ts-ignore
```

Comment:
I'm hoping to get this resolved via #1241 - in the meantime, disabling this rule is not the appropriate thing to do; instead you should use `@ts-expect-error` with a reason:

```suggestion
    // @ts-expect-error -- we don't have a static type for rule names
```

Reply:
Merged in `main`, is this still right?

Reply:
na now it can go completely